main.go

```diff
@@ -37,53 +121,41 @@ func main() {
 }
 
 func RollingBackupCommand() cli.Command {
+
+	snapshotFlags := []cli.Flag{
+		cli.DurationFlag{
+			Name:  "creation",
+			Usage: "Create backups after this time interval in minutes",
+			Value: 5 * time.Minute,
+		},
+		cli.DurationFlag{
+			Name:  "retention",
+			Usage: "Retain backups within this time interval in hours",
+			Value: 24 * time.Hour,
+		},
+		cli.BoolFlag{
+			Name:  "once",
+			Usage: "Take backup only once",
+		},
+	}
+
+	snapshotFlags = append(snapshotFlags, commonFlags...)
+
 	return cli.Command{
-		Name:   "rolling-backup",
-		Usage:  "Perform rolling backups",
-		Action: RollingBackupAction,
-		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:  "endpoints",
-				Usage: "Etcd endpoints",
-				Value: "127.0.0.1:2379",
-			},
-			cli.DurationFlag{
-				Name:  "creation",
-				Usage: "Create backups after this time interval in minutes",
-				Value: 5 * time.Minute,
-			},
-			cli.DurationFlag{
-				Name:  "retention",
-				Usage: "Retain backups within this time interval in hours",
-				Value: 24 * time.Hour,
-			},
-			cli.BoolFlag{
-				Name:   "debug",
-				Usage:  "Verbose logging information for debugging purposes",
-				EnvVar: "RANCHER_DEBUG",
-			},
-			cli.BoolFlag{
-				Name:  "once",
-				Usage: "Take backup only once",
+		Name:  "rke-tools",
+		Usage: "Perform rke tools",
+		Subcommands: []cli.Command{
+			{
+				Name:   "save",
+				Usage:  "Take snapshot on all etcd hosts and backup to s3",
+				Flags:  snapshotFlags,
+				Action: RollingBackupAction,
 			},
-			cli.StringFlag{
-				Name:  "name",
-				Usage: "Backup name to take once",
-			},
-			cli.StringFlag{
-				Name:   "cacert",
-				Usage:  "Etcd CA client certificate path",
-				EnvVar: "ETCD_CACERT",
-			},
-			cli.StringFlag{
-				Name:   "cert",
-				Usage:  "Etcd client certificate path",
-				EnvVar: "ETCD_CERT",
-			},
-			cli.StringFlag{
-				Name:   "key",
-				Usage:  "Etcd client key path",
-				EnvVar: "ETCD_KEY",
+			{
+				Name:   "restore",
```

Comment:
this subcommand is not exactly restore, as far as i understand we are saving/downloading the backup from s3